.travis.yml

```diff
@@ -43,6 +44,9 @@ install:
 script:
   # Building the recipe will run the tests too.
   - mv .git/shallow .git/shallow-not || true
+  - if [[ "${PYTHON_VERSION}" == "3.7" ]]; then
+        conda config --add channels conda-forge/label/gcc7;
```

Comment:
This should be done for all Python versions per [this comment]( https://github.com/conda-forge/python-feedstock/issues/220#issuecomment-431088554 ).